autogluon/utils/tabular/features/abstract_feature_generator.py

```diff
@@ -127,13 +92,15 @@ def fit_transform(self, X: DataFrame, y=None, banned_features=None, fix_categori
         X = X.drop(self.features_to_remove, axis=1, errors='ignore')
         self.features_init_to_keep = copy.deepcopy(list(X.columns))
         self.features_init_types = X.dtypes.to_dict()
+        self.feature_type_family_init_raw = get_type_groups_df(X)
+
         X.reset_index(drop=True, inplace=True)
         X_features = self.generate_features(X)
         for column in X_features:
             unique_value_count = len(X_features[column].unique())
             if unique_value_count == 1:
                 self.features_to_remove_post.append(column)
-            elif column in self.feature_type_family['object'] and (unique_value_count / X_len > 0.99):
+            elif 'object' in self.feature_type_family and column in self.feature_type_family['object'] and (unique_value_count / X_len > 0.99):
```

Comment:
consider adding TODO to make 0.99 here a kwarg of FeatureGenerator

Reply:
Added